src/demo-app/sticky-header/sticky-header-demo.scss

```diff
@@ -0,0 +1,88 @@
+.demo-sticky {
+  .centered {
+    text-align: center;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    height: 500px;
+    overflow: auto;}
+
+  button {
+    position: fixed;
+    margin: 16px;
+    z-index: 99;
+  }
+
+  * {
+    margin: 0;
+    padding: 0;
+  }
+
+  body {
+    font: 14px/1.5 Helvetica, Arial, sans-serif;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+
+  }
+
+  h2 {
+    font-size: 20px;
+    padding: 20px;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+  }
+
+  p {
+    margin-bottom: 1.5em;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+  }
+
+  div{
+    margin: 0; border: 0; padding: 0;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+  }
+}
+
+.mat-radio-button {
+  display: block;
+}
+
+.centered {
+  text-align: center;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  height: 500px;
+  //width: 900px;
+  overflow: auto;}
+
+.container {
+  margin: 0;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+}
+
+.rside {
+  background: white;
+  -webkit-appearance: none;
+  padding: 20px;
+}
+
+.rside1 {
+  background: #cecece;
+  -webkit-appearance: none;
```

Comment:
nit: You have lots of duplicate of `-webkit-appearance: none;padding: 20px;`. Merge them into one?